docs/docs/tutorials/product-name-scout-agent.md

```
@@ -0,0 +1,307 @@
+
+```bash
+# From the project root
+saiki --agent agents/product-name-researcher/product-name-researcher.yml
+```
+
+## Step 7: Testing with Example Prompts
+
+Let's test the agent with realistic product name research scenarios:
+
+### Basic Name Validation
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)